admin/test/features/generate_pupil_pins.feature

```diff
@@ -171,3 +171,7 @@ Feature: Generate Pupil PINs
     Given I have generated pins for all pupils in a group
     When a pupil becomes available for pin generation again
     Then I should be able to filter by groups on the generate pins page
+
```

Comment:
@sinhaamit1980 maybe add a test for when there are no check forms? Just to ensure nothing breaks?